ResearchKit/Common/ORKChoiceAnswerFormatHelper.h

```diff
@@ -41,6 +42,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (NSUInteger)choiceCount;
 
+- (id<ORKAnswerOption>)answerOptionAtIndex:(NSUInteger)index;
 - (nullable ORKImageChoice *)imageChoiceAtIndex:(NSUInteger)index;
 - (nullable ORKTextChoice *)textChoiceAtIndex:(NSUInteger)index;
```

Comment:
We should keep the functions, since especially `textChoiceAtIndex:` is used quite often throughout the framework. Valid classes for this helper are image choice, text choice, value picker and text scale answer format. All of them either use image or text choices internally. So what other convenience methods should we add?